src/application.ts

```diff
@@ -44,21 +48,14 @@ export class MyriadApiApplication extends BootMixin(
     this.configure(RestExplorerBindings.COMPONENT).to({
       path: '/explorer',
     });
-    this.component(RestExplorerComponent);
 
-    // Add cron component
-    this.component(CronComponent);
-    // Deactivate cron job for now
-    // this.add(createBindingFromClass(FetchContentSocialMediaJob))
-    // this.add(createBindingFromClass(RemovedContentJob))
-    // Optional:
-    // this.add(createBindingFromClass(FetchContentTwitterJob))
-    // this.add(createBindingFromClass(FetchContentRedditJob))
-    // this.add(createBindingFromClass(UpdatePostsJob))
+    // Bind component
+    this.bindComponent();
 
-    // Add services
-    this.service(FCMService)
-    this.service(NotificationService)
+    // Bind services
+    this.bindService();
+
+    this.add(createBindingFromClass(FetchContentSocialMediaJob));
```

Comment:
bor @abdulhakim2902 bagian ini samain aja kek this.bindComponent(); dan this.bindService();